data/json/vehicleparts/cargo.json

```
@@ -143,6 +143,13 @@
     "symbol": "o",
     "type": "vehicle_part"
   },
+  {
+    "copy-from": "basketsm_abstract",
+    "id": "basketsm_wheelchair",
+    "looks_like": "basketlg",
```

```suggestion
    "looks_like": "basketsm",
```
Since that's what it's replacing.